science/esmf/Portfile

```diff
@@ -22,7 +22,7 @@ description         software for building and coupling weather, climate, and rel
 long_description    The ESMF defines an architecture for composing complex, coupled \
                     modeling systems and includes data structures \
                     and utilities for developing individual models.
-homepage            https://www.earthsystemcog.org/projects/esmf/
+homepage            http://www.earthsystemmodeling.org
```

Comment:
please use `https`

Reply:
That website currently supports only http, not https.  Please keep this URLas http, until such time as the website is updated for https.

Reply:
this works fine for me: https://earthsystemmodeling.org/

Reply:
Okay, thank you for finding that.  Sorry I missed it the first time.  I pushed the new URL as you suggested.  Is this okay now?